### PR TITLE
Fixed incorrect references to external resources

### PR DESCRIPTION
**Pull request by nacgarg**
The un-litecoin-ification added some nonexistent links which I replaced with information on the main website.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 This issue tracker is only for technical issues related to garlicoin-core.
 
-General garlicoin questions and/or support requests and are best directed to the [garlicointalk.io forums](https://garlicointalk.io/).
+General garlicoin questions and/or support requests and are best directed to the [garlicoin subreddit](https://www.reddit.com/r/garlicoin/) or [garlicoin discord server](https://discord.gg/garlicoin).
 
-For reporting security issues, please contact the Garlicoin developers on the #garlicoin-dev Freenode IRC channel or alternatively you can email us at contact@garlicoin.org.
+For reporting security issues, please contact the Garlicoin developers on the #garlicoin-dev Freenode IRC channel or alternatively you can email us at help@garlicoin.com.
```
Comment (retosen) [inline .github/ISSUE_TEMPLATE.md]: We don't currently have a freenode channel nor do we own the garlicoin.com domain.

Comment (jdleo) [reply .github/ISSUE_TEMPLATE.md]: yeah, this was just left in from search/replace. whole section needs to be removed

**Comment by nacgarg** [comment]
Yeah, that was the info I found on the website so we should probably fix
that as well.

**Comment by nacgarg** [comment]
@retosen I removed that section

**Comment by ryan-shaw** [comment]
@nacgarg conflicts, can you rebase please

**Pull request merged**